core/vdbe/execute.rs

```diff
@@ -101,6 +101,16 @@ use super::{make_record, Program, ProgramState, Register};
 use crate::resolve_ext_path;
 use crate::{bail_constraint_error, must_be_btree_cursor, MvStore, Pager, Result};
 
+/// Macro to destructure an Insn, only to be used when it
+/// is *impossible* to be another variant.
+macro_rules! assert_insn {
```

Comment:
Can we name this something different, like get_insn or load_insn as it could misleading because it does more than just asserting, it also loads insn in the scope